src/casbin_actor.rs

```diff
@@ -29,6 +35,12 @@ pub enum CasbinResult {
     Enforce(bool),
     AddPolicy(bool),
     AddPolicies(bool),
+    AddNamedPolicy(bool),
+    AddNamedPolicies(bool),
+    AddGroupingPolicy(bool),
+    AddGroupingPolicies(bool),
+    AddNamedGroupingPolicy(bool),
+    AddNamedGroupingPolicies(bool),
```

Comment:
Why are we adding these functions? Can't the user just use the exposed enforcer instance and just call its functions?

Reply:
> Why are we adding these functions? Can't the user just use the exposed enforcer instance and just call its functions?

@hsluoyz  Yes, user can expose the enforcer and call its function. But in actix-web, actor way will be a more safer and more efficient way to call casbin function. The actor will live all along the actix-web app so user can get it for casbin uses.